client/lobby/RandomMapTab.h

```diff
@@ -24,7 +25,7 @@ class CToggleButton;
 class CLabel;
 class CLabelGroup;
 
-class RandomMapTab : public CIntObject
+class RandomMapTab : public InterfaceBuilder
```

Comment:
It is a bit weird to have GUI element inherit from class named InterfaceBuilder.
Personally if I see something named "Builder" I would presume that this is factory-like class that creates objects one way or another.
Proposition:
a) rename "InterfaceBuilder" to... let's say "BuildableWindow" or "BuildableIntObject" (or any other suitable name)
b) if this makes sense, split InterfaceBuilder class into "builder" that processes config and "object" that is created in "build" process.